subsys/sd/sdmmc.c

```diff
@@ -998,6 +998,11 @@ int sdmmc_card_init(struct sd_card *card)
 			ocr_arg |= SD_OCR_VDD29_30FLAG;
 		}
 		ocr_arg |= (SD_OCR_VDD32_33FLAG | SD_OCR_VDD33_34FLAG);
+		/* Momentary delay before initialization OCR. Some cards will
+		 * never leave busy state if init OCR is sent too soon after
+		 * probing OCR
+		 */
+		k_busy_wait(100);
```

Comment:
Can you archive the same using `k_usleep(int32_t us);`?

Reply:
We probably could using `k_usleep`, yes. In my mind the delay here is short enough that k_busy_wait is acceptable, and we don't really want to yield to another thread here, hence why I chose `k_busy_wait`